src/Integration/Suppression/SonarQubeIssuesProvider.cs

```diff
@@ -103,6 +118,12 @@ private string BuildModuleKey(string projectGuid)
 
         private async Task SynchronizeSuppressedIssues()
         {
+            // TODO: may not be connected
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)